amundsen_application/base/base_announcement_client.py

```diff
@@ -31,20 +32,16 @@ def _create_error_response(message: str) -> Response:
             return make_response(payload, HTTPStatus.INTERNAL_SERVER_ERROR)
 
         try:
```

Comment:
i removed this outer try/except. There's nothing it catches which is not caught by the inner try/except.